src/styles/_normalize.scss

```diff
@@ -0,0 +1,5 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
don't use `*` to reset something, just for `box-sizing`. To reset something list all tags